src/conventional/amv_ssec_ascii2ioda.py

```diff
@@ -0,0 +1,323 @@
+                satid, unk_sat = get_id(row['sat'], unk_sat)
+                data['satelliteID'] = np.append(data['satelliteID'], satid)
+                data['sensorCentralFrequency'] = np.append(data['sensorCentralFrequency'], freq)
+                data['sensorZenithAngle'] = np.append(data['sensorZenithAngle'], float(row['rff']))
+                data['windTrackingCorrelation'] = np.append(data['windTrackingCorrelation'], float(row['qi']))
+                data['windHeightAssignMethod'] = np.append(data['windHeightAssignMethod'], int(row['int']))
+
+                pres = float(row['pre'])*100.
+                data['air_pressure'] = np.append(data['air_pressure'], pres)
+                wdir = float(row['dir'])*1.0
+                wspd = float(row['spd'])*1.0
+                if (wdir >= 0 and wdir <= 360 and wspd >= 0 and wspd < 300):
+                    uwnd, vwnd = meteo_utils.meteo_utils().dir_speed_2_uv(wdir, wspd)
+                else:
+                    uwnd = float_missing_value
+                    vwnd = float_missing_value
+
+                data['eastward_wind'] = np.append(data['eastward_wind'], uwnd)
+                data['northward_wind'] = np.append(data['northward_wind'], vwnd)
+
+            except KeyError as e:
+                if not keyerr:
+                    logging.warning(file_name + ' is missing variable ' + e.args[0])
+                keyerr = True
+
+                if (e.args[0] == 'dir') | (e.args[0] == 'spd'):
+                    data['eastward_wind'] = np.append(data['eastward_wind'], float_missing_value)
+                    data['northward_wind'] = np.append(data['northward_wind'], float_missing_value)
+                else:
+                    outname, missing = get_outname(e.args[0])
+                    data[outname] = np.append(data[outname], missing)
+
+        for f in unk_freq:
+            logging.warning(file_name + ' contains unknown frequency ' + f)
+
+        for s in unk_sat:
+            logging.warning(file_name + ' contains unknown satellite ID ' + s)
+
+    return data
+
+
+def get_frequency(obs_type, unk_freq):
+
+    if obs_type == 'IR':
+        freq = 2.99792458E+14/10.7
+    elif (obs_type == 'WVCA') | (obs_type == 'WVCT') | (obs_type == 'WV'):
+        freq = 2.99792458E+14/6.7
+    elif (obs_type == 'SWIR'):
+        freq = 2.99792458E+14/3.9
+    elif (obs_type == 'VIS'):
+        freq = 2.99792458E+14/0.65
+    else:
+        unk_freq.append(obs_type)
+        freq = float_missing_value
+
+    return freq, unk_freq
+
+
+def get_id(sat_name, unk_sat):
+
+    if sat_name == 'HMWR08':
+        satid = 173
+    elif sat_name == 'MET11':
+        satid = 70
+    elif sat_name == 'MET8':
+        satid = 55
+    elif sat_name == 'GOES16':
+        satid = 270
+    elif sat_name == 'GOES17':
+        satid = 271
+    else:
+        unk_sat.append(unk_sat)
+        satid = int_missing_value
+
+    return satid, unk_sat
+
+
+def get_outname(key):
+
+    if key == 'type':
```

Comment:
Similar to the prior comment, I believe we can just list the missing data type together with the variable name near the very top of the code in the Dictionary that is already present at the top.  Otherwise, it is too easy to miss changing in more than one place.  I spent considerable time in editing the 1st version of code to keep all JEDI Data Conventions information at the very top of the code - making it super simple to locate and change rather than multiple places in code.